packages/nc-gui/components/general/Overlay.vue

```diff
@@ -11,6 +11,7 @@ interface Props {
   target?: TeleportProps['to']
   teleportDisabled?: TeleportProps['disabled']
   transition?: boolean
+  className?: string
```

Comment:
You don't need an extra prop for class 😄 

```html
<div>
	 <div v-bind="$attrs">Foo</div>
</div>
```

$attrs will contain the `class` attribute already so they're bound to the div in the overlay.

Technically the prop you added does nothing right now ^^

Reply:
Habits from react 😅